events/population_grow.txt

```diff
@@ -465,57 +498,113 @@ province_event = {
-		}
-		modifier = {
-			factor = 0.95 	# base value, almost untouched. Similar to modern age Europe (late 1600s)
-			check_variable = { PG_trb_grow = 100 }	NOT = { check_variable = { PG_trb_grow = 130 } }
-		}
-		modifier = {
-			factor = 0.65	# 0.85% yearly, early 19th Century France (slow demographic explosion). // check: 1/(8*360/12*0.65) = 0.641%
-			check_variable = { PG_trb_grow = 130 }	NOT = { check_variable = { PG_trb_grow = 160 } }
-		}
-		modifier = {
-			factor = 0.4	# 1,35%, 1890s UK (demographic explosion)
-			check_variable = { PG_trb_grow = 160 }	NOT = { check_variable = { PG_trb_grow = 200 } }
-		}
-		modifier = {
-			factor = 0.2 	# 2,7%, usual growth of India 1970s
-			check_variable = { PG_trb_grow = 200 }	NOT = { check_variable = { PG_trb_grow = 500 } }
+	option = {
+		name = "migrations.1.1"
+
+		every_owned_province = {
+			limit = { is_city = yes }
+			if = {
+				limit = {
+					base_manpower = 1
+					check_variable = { PG_trb_grow = 100 }	# Added, since it seems growth was getting out of hand even adding a factor 30 to <0.8
+				}
+				random_list = {
+					29 = {
+						# See exact calculation of growth below, on the Inmediate part. Basically it must fire this event SIX times if base tribal population is 1, and thus the total mtth for a total doubling of population is 6 x 360 months = 2.160 months = 180 years. 1/180 = base yearly growth rate of 0.55%.
+						# There will be a help for the first pop to double, as it would destroy the player's patience
+
+						modifier = {
+							factor = 0.2 	# Needs a kickstart for those at only 1 tribal. Starting at 2.0 growth
+							check_variable = { PG_trb_grow = 200 }
+							base_manpower = 1	NOT = { base_manpower = 2 }
+						}
+						modifier = {
+							factor = 0.95 	# base value, almost untouched. Similar to modern age Europe (late 1600s)
+							check_variable = { PG_trb_grow = 100 }	NOT = { check_variable = { PG_trb_grow = 130 } }
+						}
+						modifier = {
+							factor = 0.65	# 0.85% yearly, early 19th Century France (slow demographic explosion). // check: 1/(8*360/12*0.65) = 0.641%
+							check_variable = { PG_trb_grow = 130 }	NOT = { check_variable = { PG_trb_grow = 160 } }
+						}
+						modifier = {
+							factor = 0.4	# 1,35%, 1890s UK (demographic explosion)
+							check_variable = { PG_trb_grow = 160 }	NOT = { check_variable = { PG_trb_grow = 200 } }
+						}
+						modifier = {
+							factor = 0.2 	# 2,7%, usual growth of India 1970s
+							check_variable = { PG_trb_grow = 200 }	NOT = { check_variable = { PG_trb_grow = 500 } }
+						}
+						modifier = {
+							factor = 0.1 	# 5,5% yearly growth, based in sub-saharan africa 1990s, or Ireland before the potato pandemic. Doubles every 24 years
+							check_variable = { PG_trb_grow = 500 }
+
+						}
+					}
+					1 = {
+						province_event = { id = population_grow.4 }
+					}
+				}
+			}
+			if = {
+				limit = {  check_variable = { PG_com_grow = 100 } }
+				random_list = {
+					44 = {

```

Comment:
why 44 and not 29, like above?

Reply:
I just converted the MTTH from the original event into random_list. They had different MTTHs so they get different random_list base cases